examples/c_code/main.c

```diff
@@ -0,0 +1,50 @@
+#include "individual.h"
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+
+double target(double x_0, double x_1) {
```

Comment:
```suggestion
double target(const double x_0, const double x_1) {
```
i think it's good practice to try to make things reasonably `const` ;)